code/datums/api.dm

```diff
@@ -633,10 +633,48 @@ proc/api_update_command_database()
 		data = null
 		return 1
 
+//Get Server Status
+/datum/topic_command/get_serverstatus_public
+	name = "get_serverstatus_public"
+	description = "Gets the serverstatus"
+	no_auth = 1
+
+/datum/topic_command/get_serverstatus/run_command(queryparams)
+	var/list/s[] = list()
```

Comment:
Minor nitpick: this should be `var/list/s = list()` or `var/s[]` (preferably former), not a mix of the two.